doc/source/ray-observability/ray-metrics.rst

```diff
@@ -261,6 +243,26 @@ To do that, you may want to customize the port that metrics gets exposed to a pr
 
 Now, you can scrape Ray's metrics using Prometheus via ``<ip>:8080``.
 
+Alternate Prometheus host location
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can choose to run Prometheus on a non-default port or on a different machine. When doing so, you should
+make sure that prometheus can scrape the metrics from your ray nodes following instructions :ref:`here <multi-node-metrics>`.
```

Comment:
Can you actually link to the exact section? Grafana section from th monitoring-and-observability.rst.

Reply:
also can you mention the default host ?